src/main/java/bio/terra/cda/app/generators/MutationSqlGenerator.java

```diff
@@ -2,7 +2,8 @@
 
 import bio.terra.cda.generated.model.Query;
 
-@EntityGeneratorData(entity = "somatic_mutation", hasFiles = false, defaultOrderBy = "case_barcode",
+// TODO - case_barcode may need to be altered
+@EntityGeneratorData(entity = "mutation", hasFiles = false, defaultOrderBy = "case_barcode",
```

Comment:
OPTIONAL: `defaultOrderBy = integer_id_alias` might save us a bit of sorting time

Reply:
Updated